### PR TITLE
react-native-animatable  - remove dep

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
react-native-animatable - remove dependency.

## Changelog
react-native-animatable - remove dependency.

### CODE REVIEW

File: generatedTypes/lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/KeyboardTrackingView.ios.d.ts [hunk 1]
```diff
@@ -13,7 +13,6 @@ import { KeyboardTrackingViewProps } from './index';
  */
 declare class KeyboardTrackingView extends PureComponent<KeyboardTrackingViewProps> {
     static displayName: string;
-    /** V6 should be change to default false */
```
Comment (ethanshar) [inline generatedTypes/lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/KeyboardTrackingView.ios.d.ts:16]: This doesn't seem relevant to this PR? 
Did you already change the useSafeArea default on another PR?

**Review by ethanshar — APPROVED** [review]

File: generatedTypes/lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/KeyboardTrackingView.ios.d.ts (hunk 1 above)
Comment (Inbal-Tish) [reply generatedTypes/lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/KeyboardTrackingView.ios.d.ts:16]: Yes. It's already merged
